controllers/controller.js

```diff
@@ -45,3 +43,23 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  const articleIdNumber = parseInt(article_id, 10);
+
```

Comment:
You will need some logic here to pass the test that you missed out. You will need to check the article exists before you invoke the fetchComments model